examples/resources/adfs-connector.yaml

```diff
@@ -5,6 +5,14 @@ version: v2
 metadata:
   name: "adfs_connector"
 spec:
+  # display allows to set the caption of the "login" button
+  # in the Web interface
+  display: "Login with SSO"
+
+  # "adfs" provider setting tells Teleport that this SAML connector uses ADFS
+  # as a provider
```

Comment:
This field is required for ADFS to override the default C14N11 canonicalizer and use the C14N10 canonicalizer.